cmds/sos/sos.go

```diff
@@ -14,11 +14,14 @@ var (
 	Registry = make(map[string]uint)
 )
 
-func read(serviceName string) (port uint, exists bool) {
+func read(serviceName string) (uint, error) {
 	RWLock.RLock()
 	defer RWLock.RUnlock()
-	port, exists = Registry[serviceName]
-	return
+	port, exists := Registry[serviceName]
+	if !exists {
+		return 0, fmt.Errorf("%v is not in the registry", serviceName)
```

Comment:
could you just return os.ErrNotExist?

Reply:
hmm, it returns "file does not exist'. That does not seems right for our use case.